File: examples-sdk/csharp/017-basic-production-fullstack/Main.cs
// 017-basic-production-fullstack — C# SDK equivalent
// Compile: vil compile --from csharp --input 017-basic-production-fullstack/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("production-fullstack", 8080);
var fullstack = new ServiceProcess("fullstack");
fullstack.Endpoint("GET", "/stack", "stack_info");
fullstack.Endpoint("GET", "/config", "full_config");
fullstack.Endpoint("GET", "/sprints", "sprints");
fullstack.Endpoint("GET", "/middleware", "middleware_info");
server.Service(fullstack);
var admin = new ServiceProcess("admin");
admin.Endpoint("GET", "/config", "full_config");
server.Service(admin);
var root = new ServiceProcess("root");
server.Service(root);
server.Compile();

File: examples-sdk/swift/017-basic-production-fullstack/main.swift
// 017-basic-production-fullstack — Swift SDK equivalent
// Compile: vil compile --from swift --input 017-basic-production-fullstack/main.swift --release

let server = VilServer(name: "production-fullstack", port: 8080)
let fullstack = ServiceProcess(name: "fullstack")
fullstack.endpoint(method: "GET", path: "/stack", handler: "stack_info")
fullstack.endpoint(method: "GET", path: "/config", handler: "full_config")
fullstack.endpoint(method: "GET", path: "/sprints", handler: "sprints")
fullstack.endpoint(method: "GET", path: "/middleware", handler: "middleware_info")
server.service(fullstack)
let admin = ServiceProcess(name: "admin")
admin.endpoint(method: "GET", path: "/config", handler: "full_config")
server.service(admin)
let root = ServiceProcess(name: "root")
server.service(root)
server.compile()
